contrib/emicklei/go-restful/example_test.go

```diff
@@ -0,0 +1,42 @@
+package restful_test
+
+import (
+	"log"
+	"net/http"
+
+	"github.com/emicklei/go-restful"
+	restfultrace "gopkg.in/DataDog/dd-trace-go.v1/contrib/emicklei/go-restful"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
+)
+
+// To start tracing requests, add the trace filter to your go-restful router.
+func Example() {
+	// create new go-restful service
+	ws := new(restful.WebService)
+
+	// instrument the service with trace
+	ws.Filter(restfultrace.Filter)
+
+	// set endpoint
+	ws.Route(ws.GET("/hello").To(
+		func(request *restful.Request, response *restful.Response) {
+
```

Comment:
Let's do something here. This doesn't give for a very good example otherwise 😄

Reply:
sure